src/vm/amd64/PInvokeStubs.asm

```diff
@@ -11,12 +11,13 @@
 include AsmMacros.inc
 include AsmConstants.inc
 
-PInvokeStubForHostWorker        equ ?PInvokeStubForHostWorker@@YAXKPEAX0@Z
-extern PInvokeStubForHostWorker:proc
-
 extern GenericPInvokeCalliStubWorker:proc
 extern VarargPInvokeStubWorker:proc
 
+ifdef FEATURE_INCLUDE_ALL_INTERFACES
```

Comment:
we otherwise need to implement dummy functions for arm64 as there are references in VM which call these functions. These are not required for coreclr. So I put the places where these are called under FEATURE_INCLUDE_ALL_INTERFACES and the asm functions under correct ifdef for all architecture.